redbot/core/utils/views.py

```diff
@@ -431,3 +431,147 @@ async def auth_button(self, interaction: discord.Interaction, button: discord.Bu
         return await interaction.response.send_modal(
             SetApiModal(self.default_service, self.default_keys)
         )
+
+
+class ConfirmView(discord.ui.View):
+    """
+    A simple ``discord.ui.View`` used for confirming something.
+
+    Parameters
+    ----------
+    author: Optional[discord.abc.User]
+        The user who you want to be interacting with the confirmation.
+        If this is omitted anyone can click yes or no.
+    timeout: float
+        The timeout of the view in seconds. Defaults to ``180`` seconds.
+    disable_buttons: bool
+        Whether to disable the buttons instead of removing them from the message after the timeout.
+         Defaults to ``False``.
```

Comment:
This may seem like a nitpick but this is actually a very important fix to how Sphinx ends up displaying this:
```suggestion
        Defaults to ``False``.
```
![image](https://github.com/Cog-Creators/Red-DiscordBot/assets/6032823/9d54f98b-1810-4f20-87a8-19fdaed9fc42)